doc/releases/mimic.rst

```diff
@@ -162,30 +162,33 @@ Upgrading from pre-Luminous releases (like Jewel)
 
 You *must* first upgrade to Luminous (12.2.z) before attempting an
 upgrade to Mimic.
-     
-Notable Changes
----------------
+
+Upgrade compatibility notes
+---------------------------
+
+These changes occurred between the Luminous and Mimic releases.
 
 * *core*:
 
-  - The `pg force-recovery` command will not work for erasure-coded PGs when a
+  - The ``pg force-recovery`` command will not work for erasure-coded PGs when a
     Luminous monitor is running along with a Mimic OSD. Please use the
     recommended upgrade order of monitors before OSDs to avoid this issue.
 
   - The sample ``crush-location-hook`` script has been removed. Its output is
     equivalent to the built-in default behavior, so it has been replaced with an
     example in the CRUSH documentation.
 
-  - The -f option of the rados tool now means "--format" instead of "--force",
-    for consistency with the ceph tool.
+  - The `-f` option of the rados tool now means `--format` instead
```

Comment:
double backticks for --format and force maybe

Reply:
updated.  really need to sort out what the rules are for this, though.. i was using "it is a shell command line thing" for double ticks and single ticks for everything else, but there is probably a real rule we should be using?

Reply:
Note that single backticks in reStructuredText have a different meaning than in Markdown - in the latter, they mean "print this on monospace", which is usually used for highlighting shell commands or commandline options in text blocks.

In [reST markup](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup), you would use double backticks to achieve the same formatting - single backticks have a special meaning there.